### PR TITLE
Add "Login Manager Settings"/gdm-settings (io.github.realmazharhussain.GdmSettings)

### PR DESCRIPTION
**Pull request by realmazharhussain**
### I confirm my submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions.
- [x] All assets referenced in the manifest are redistributable by any party.
- [x] I am an upstream contributor to the project.
- [x] The domain (GitHub) has a policy for delegating subdomains.
- [x] No additional patches or files exist.

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: io.github.realmazharhussain.GdmSettings.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  - '*.la'
+  - '*.a'
+modules:
+  - name: blueprint-compiler
+    builddir: true
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/jwestman/blueprint-compiler.git
+        branch: main
```
Comment (hfiguiere) [inline io.github.realmazharhussain.GdmSettings.yaml:33]: you need an explicit revision, not a branch

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
bot, build io.github.realmazharhussain.GdmSettings

File: io.github.realmazharhussain.GdmSettings.yaml (hunk 1 above)
Comment (realmazharhussain) [reply io.github.realmazharhussain.GdmSettings.yaml]: Done now.

File: io.github.realmazharhussain.GdmSettings.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+runtime: org.gnome.Platform
+runtime-version: "42"
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --talk-name=org.freedesktop.Flatpak
+  - --filesystem=host-os
+  - --filesystem=host-etc
```
Comment (hfiguiere) [inline io.github.realmazharhussain.GdmSettings.yaml:13]: why does it need these filesystem permission and why does it need to talk to the flatpak dbus?

Comment (realmazharhussain) [reply io.github.realmazharhussain.GdmSettings.yaml:13]: So this application is kind of a system application.
## Flatpak DBus
It needs to run a script it creates with escalated privileges on the host.
## --filesystem=host-os
It needs access to the system themes which are located in `/usr/share` and `/usr/local/share` on the host and it also needs access to `/usr/share/gnome-shell` directory.
## --filesystem=host-etc
It needs access to `/etc/dconf/db/gdm.d` and `/etc/dconf/profile/gdm`. Now that I think about it, this permission can be minimized. I'm gonna do it. Wait a second.

Comment (realmazharhussain) [reply io.github.realmazharhussain.GdmSettings.yaml]: Done
`--filesystem=host-etc` replaced with specific permissions
```
--filesystem=/etc/dconf/profile/gdm
--filesystem=/etc/dconf/db/gdm.d
```

**Comment by barthalion** [comment]
bot, build io.github.realmazharhussain.GdmSettings

**Comment by realmazharhussain** [comment]
Anything else that I need to do?

**Pull request closed without merging**